Modules/Scripted/SampleData/SampleData.py

```diff
@@ -289,10 +298,22 @@ def registerBuiltInSampleDataSources(self):
           ('CTBrain', 'MRBrainT1', 'MRBrainT2')),
         )
 
-    if not slicer.modules.sampleDataSources.has_key('BuiltIn'):
-      slicer.modules.sampleDataSources['BuiltIn'] = []
+    if not slicer.modules.sampleDataSources.has_key(self.builtInCategoryName):
+      slicer.modules.sampleDataSources[self.builtInCategoryName] = []
     for sourceArgument in sourceArguments:
-      slicer.modules.sampleDataSources['BuiltIn'].append(SampleDataSource(*sourceArgument))
+      slicer.modules.sampleDataSources[self.builtInCategoryName].append(SampleDataSource(*sourceArgument))
+
+  def registerDevelopmentSampleDataSources(self):
+    iconPath = os.path.join(os.path.dirname(__file__).replace('\\','/'), 'Resources','Icons')
+    self.registerCustomSampleDataSource(
+      category=self.developmentCategoryName, sampleName='TinyPatient',
+      uris=['http://slicer.kitware.com/midas3/download/item/245205/TinyPatient_CT.nrrd',
+            'http://slicer.kitware.com/midas3/download/item/367020/TinyPatient_Structures.seg.nrrd'],
+      fileNames=['TinyPatient_CT.nrrd', 'TinyPatient_Structures.seg.nrrd'],
+      nodeNames=['TinyPatient_CT', 'TinyPatient_Structures'],
```

Comment:
TinyPatient_Segments?

Reply:
Yes, a body and a tumor. Everything is super low resolution for testing

Reply:
Oh I see, the name... OK :)